### ISSUE TITLE
Question: Why does BuildingElement not have any geometry?

### ISSUE THREAD

**Issue by filipGG** [RealEstateCore/rec#262]
In my case I would like to be able to represent a wall as either a series of lines or as a polygon for thick walls.

I could store it as a custom property, but in my mind having actual geometry would make more sense.
What do you think?

**Comment by ogiel** [comment]
I agree that this would make sense to have (and something to want). Just checked and it is indeed not possible to create a geometry twin and give it a relationship to a building element (no relationships available). 

Maybe make a PR that contains relationship between building element and geometry equally to relationship between architecture and geometry, and then create building element + geometry twin pairs for each element.
